pkgs/by-name/un/unsilence/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, ffmpeg
+,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "unsilence";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "lagmoellertim";
+    repo = "unsilence";
+    rev = version;
+    sha256 = "sha256-M4Ek1JZwtr7vIg14aTa8h4otIZnPQfKNH4pZE4GpiBQ=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    rich
+    pythonRelaxDepsHook
+  ];
+
+  propagatedBuildInputs = [
+    python3Packages.rich
+    python3Packages.setuptools # imports pkg_resources.parse_version
+  ];
+
+  makeWrapperArgs = [
+    "--suffix PATH : ${lib.makeBinPath [ ffmpeg ]}"
```

Comment:
moved ffmpeg from propagatedBuildInputs to the wrapper because the binaries are used